sn_aia_agent_config_8c0b5e79378422100d1bdfc734924b3d.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update sys_domain="global" table="sn_aia_agent_config">
```

Comment:
The following issues were found in the file:
- **1**: The `sys_name` tag is empty.